domain/src/main/kotlin/no/nav/su/se/bakover/domain/behandling/avslag/AvslagManglendeDokumentasjon.kt

```diff
@@ -0,0 +1,79 @@
+import no.nav.su.se.bakover.domain.søknadsbehandling.LukketSøknadsbehandling
+import no.nav.su.se.bakover.domain.søknadsbehandling.Stønadsperiode
+import no.nav.su.se.bakover.domain.søknadsbehandling.Søknadsbehandling
+import java.time.Clock
+import java.time.LocalDate
+
+data class AvslagManglendeDokumentasjon private constructor(
+    val søknadsbehandling: Søknadsbehandling.Iverksatt.Avslag.UtenBeregning,
+) : ErAvslag {
+    override val avslagsgrunner: List<Avslagsgrunn> = listOf(Avslagsgrunn.MANGLENDE_DOKUMENTASJON)
+
+    companion object {
+        fun tryCreate(
+            søknadsbehandling: Søknadsbehandling,
+            saksbehandler: NavIdentBruker.Saksbehandler,
+            fritekstTilBrev: String = "",
+            clock: Clock = Clock.systemUTC(),
+        ): Either<SøknadsbehandlingErIUgyldigTilstand, AvslagManglendeDokumentasjon> {
+            return when (søknadsbehandling) {
+                is Søknadsbehandling.Iverksatt,
+                is LukketSøknadsbehandling,
+                is Søknadsbehandling.TilAttestering,
+                -> {
+                    SøknadsbehandlingErIUgyldigTilstand.left()
+                }
+                else -> {
+                    AvslagManglendeDokumentasjon(
+                        søknadsbehandling = Søknadsbehandling.Iverksatt.Avslag.UtenBeregning(
+                            id = søknadsbehandling.id,
+                            opprettet = søknadsbehandling.opprettet,
+                            sakId = søknadsbehandling.sakId,
+                            saksnummer = søknadsbehandling.saksnummer,
+                            søknad = søknadsbehandling.søknad,
+                            oppgaveId = søknadsbehandling.oppgaveId,
+                            behandlingsinformasjon = søknadsbehandling.behandlingsinformasjon,
+                            fnr = søknadsbehandling.fnr,
+                            saksbehandler = saksbehandler,
+                            attesteringer = søknadsbehandling.attesteringer.leggTilNyAttestering(
+                                Attestering.Iverksatt(
+                                    attestant = NavIdentBruker.Attestant(saksbehandler.navIdent),
+                                    opprettet = Tidspunkt.now(clock),
+                                ),
+                            ),
+                            fritekstTilBrev = fritekstTilBrev,
+                            /**
+                             * Setter stønadsperiode til inneværende måned dersom det ikke eksisterer.
+                             * Workaround for å oppfylle krav om dette feltet, selv om det har liten praktisk
+                             * betydning for avslag.
+                             */
+                            stønadsperiode = søknadsbehandling.stønadsperiode ?: Stønadsperiode.create(
+                                periode = Periode.create(
+                                    fraOgMed = LocalDate.now(clock).startOfMonth(),
```

Comment:
Gir det mer mening å bruke søknaden sin mottattdato?

Reply:
Det kan godt tenkes, føles noe "kunstig" uansett hva vi går for her.. bare en verdi vi må oppfylle.